tests/Bridge/Laravel/SlugifyProviderTest.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Cocur\Slugify\Bridge\Laravel;
+namespace Cocur\Slugify\Bridge\Tests\Laravel;
```

Comment:
This is not consistent with the other changes you made. Shouldn't it be `Tests\Bridge\Laravel`?